android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -1744,6 +1752,7 @@ private WritableMap buildRoom(Room currentRoom) {
         roomMap.putMap("dominantSpeaker", currentRoom.getDominantSpeaker() != null ? buildParticipantWithTracks(currentRoom.getDominantSpeaker()) : null);
         roomMap.putArray("remoteParticipants", buildRemoteParticipants(currentRoom.getRemoteParticipants()));
         roomMap.putMap("localParticipant", buildParticipantWithTracks(currentRoom.getLocalParticipant()));
+        roomMap.putString("localParticipantRegion", currentRoom.getLocalParticipant().getSignalingRegion());
```

Comment:
I would suggest renaming the parameter to `signalingRegion`
```suggestion
        roomMap.putString("signalingRegion", currentRoom.getLocalParticipant().getSignalingRegion());
```

Reply:
This applies to iOS and JS sources as well